rct229/rulesets/ashrae9012019/ruleset_functions/get_zone_peak_internal_load_floor_area_dict.py

```diff
@@ -0,0 +1,89 @@
+from rct229.rulesets.ashrae9012019.ruleset_functions.baseline_systems.baseline_system_util import (
+    find_exactly_one_zone,
+    find_exactly_one_schedule,
+)
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import ZERO
+
+
+def get_zone_peak_internal_load_floor_area_dict(rmi, zone_id):
+    """
+    Finds the peak coincident internal loads of a zone and returns the value in btu/h/ft2
+    The function returns a dict giving 2 values: {"PEAK":total peak btu/h/ft2 in the zone, "AREA":total zone area} the
+    total peak btu/sf is the internal non-coincident peak loads in all spaces in the zone
+
+    The function does not raise exception for missing values rather it applies default values.
+
+    Parameters
+    ----------
+    rmi: dict
+    A dictionary representing a RuleModelInstance object as defined by the ASHRAE229 schema
+    zone_id: string
+    zone id
+
+    Returns
+    -------
+    result: dict
+    a dictionary that contains two keys, peak, and area.
+    """
+    zone = find_exactly_one_zone(rmi, zone_id)
+    zone_area = ZERO.AREA
+    zone_load = ZERO.POWER
+
+    for space in find_all("$.spaces[*]", zone):
+        space_area = space.get("floor_area", ZERO.AREA)
+        zone_area += space_area
+        for light in find_all("$.interior_lighting[*]", space):
+            lighting_design_schedule = find_exactly_one_schedule(
+                rmi,
+                getattr_(light, "interior_lighting", "lighting_multiplier_schedule"),
```

Comment:
The notes for lighting_multiplier_schedule says:
"If the schedule is not present, the multiplier is always one. Constraint to use when implemented :Schedule:"

Does this mean that instead of throwing an exception if _lighting_multiplier_schedule_ is missing we should instead  take it to be a schedule with all ones?

Reply:
You are correct. Updated this code